.github/pull_request_template.md

```diff
@@ -1,40 +1,26 @@
 ## Overview
```

Comment:
```suggestion
```

Reply:
Just a thought that this feels unnecessary because people have been confused about the difference between "Overview" and "Description"